RFC/0001-lightweight-rfc-process.md

```diff
@@ -0,0 +1,117 @@
+# RFC 0001: Lightweight RFC Process for IPFS Specifications
+
+- Start Date: 2022-06-10
+- Related Issues:
+  - [ipfs/specs/issues/286](https://github.com/ipfs/specs/issues/286)
+
+# Summary
+
+This Request for Change (RFC) introduces lightweight RFC process
+for IPFS specifications [repository][1].
+
+[1]: https://github.com/ipfs/specs/
+
+# Motivation
+
+Protocol design discussions often take place in a repository of an IPFS
+implementation. Often lacking useful input from IPFS Stewards, other
+implementations, service operators and the wider IPFS Community.
+
+IPFS Project needs a mechanism for proposing and evaluating specification
+improvements that are not tied to a specific programming language
+or implementation of IPFS.
+
+# Detailed design
+
+Adopt an informal RFC process for [ipfs/specs][1] repository, providing
+minimal structure for opening, reviewing, and merging specification changes.
+
+## Opening
+
+Changes to IPFS specifications can be proposed by opening a PR against the
+repository, and including a short "request for change" document based on the
+template in `RFC/0000-template.md`.
+
+## Reviewing
+
+IPFS Stewards will review new RFC PRs during weekly triage.
```

Comment:
Do you want to explicitly list the github handles of the Stewards? Or punt until later.

Reply:
Punt. This is one of the topics I hope to flesh during our meetup in July and beyond, after we get some experience with RFC process.

For now, we will be reusing the existing [`@ipfs/w3dt-stewards`](https://github.com/orgs/ipfs/teams/w3dt-stewards/members)  team.

Reply:
Good idea. Created  @ipfs/specs-stewards  – figuring out who should stay/be added is discussed in https://github.com/ipfs/specs/issues/291